.github/workflows/backend_deploy_prod.yaml

```
@@ -0,0 +1,170 @@
+name: Prod backend build and deploy
+
+concurrency:
+  group: prod_deploy
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - master
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: lubimovka_backend-1
+  DEPLOY_PATH: /usr/local/src
+
+defaults:
+  run:
+    working-directory: .
+
+jobs:
+  build-and-push-image-to-github-packages:
+    name: Push Docker image to GitHub Packages
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: master
+      -
+        name: Docker login
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      -
+        name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: Dockerfile
+          labels: runnumber=${GITHUB_RUN_ID}
+          push: true
+          tags: ${{ env.REGISTRY }}/${{ github.actor }}/${{ env.IMAGE_NAME }}:master,${{ env.REGISTRY }}/${{ github.actor }}/${{ env.IMAGE_NAME }}:latest,${{ env.REGISTRY }}/${{ github.actor }}/${{ env.IMAGE_NAME }}:${{ github.sha }}
```

А можно как-то так сделать?
```suggestion
          tags: 
              ${{ env.REGISTRY }}/${{ github.actor }}/${{ env.IMAGE_NAME }}:master,
              ${{ env.REGISTRY }}/${{ github.actor }}/${{ env.IMAGE_NAME }}:latest,
              ${{ env.REGISTRY }}/${{ github.actor }}/${{ env.IMAGE_NAME }}:${{ github.sha }}
```
И в других местах, так читается лучше